src/components/authorization/policies/by-role/field-partner.policy.ts

```diff
@@ -1,6 +1,29 @@
-import { member, Policy, Role, variant } from '../util';
+import { inherit, member, Policy, Role, variant } from '../util';
 
 @Policy(Role.FieldPartner, (r) => [
+  r.Ceremony.read,
+  inherit(
+    r.Engagement.read,
+    r.InternshipEngagement.when(member).edit.specifically(
+      (p) => p.ceremony.none
+    ),
+    r.LanguageEngagement.when(member).read
+  ),
```

Comment:
They can read LanguageEngagement ceremonies but not Internship ones?

Reply:
Should be both; a miss.  👍🏼

Reply:
Talked to business...both should be read only.